lang/ilias_de.lang

```diff
@@ -1110,6 +1110,7 @@ assessment#:#question_is_part_of_running_test#:#Die Frage ist Teil eines laufend
 assessment#:#question_marking#:#Fragen markieren
 assessment#:#question_marking_description#:#Teilnehmer erhalten die Möglichkeit, Fragen für sich zu markieren. Die Markierungen werden auch in der 'Übersicht Testdurchlauf' angezeigt, sofern diese aktiviert ist.
 assessment#:#question_not_answered#:#Die Frage wurde nicht beantwortet
+assessment#:#question_pool_title#:#Question Pool Titel
```

Comment:
Sollte "Titel Fragenpool" heißen, oder?

Reply:
Im Featurewiki steht keine Deutsche Übersetzung. Beim Englischen steht aber "Question Pool Title" weshalb "Question Pool Titel" sich, meiner Meinung nach, am besten eignet :)

Reply:
I replaced the label and the description column can now be hidden but is shown by default.